website/versioned_docs/version-0.60/typescript.md

```diff
@@ -162,7 +162,7 @@ You can explore the syntax more in the [TypeScript playground][tsplay].
 
 - [TypeScript Handbook][ts-handbook]
 - [React's documentation on TypeScript][react-ts]
-- [React + TypeScript Cheatsheets][cheat] has a good overview on how to use React with TypeScript
+- [React + TypeScript Cheatsheets][cheats] has a good overview on how to use React with TypeScript
```

Comment:
This also looks like it should not be a part of this PR

Reply:
#1622